src/test/resources/jsonld/HT019268984

```diff
@@ -0,0 +1,114 @@
+    "dateModified" : "20170327",
+    "id" : "http://lobid.org/resources/HT019268984",
+    "label" : "lobid Ressource"
+  },
+  "extent" : "467 Seiten",
+  "hasItem" : [ {
+    "id" : "http://lobid.org/items/HT019268984:DE-Kn28:Fag%202080%20(1)#!",
+    "label" : "lobid Bestandsressource"
+  } ],
+  "hbzId" : "HT019268984",
+  "id" : "http://lobid.org/resources/HT019268984#!",
+  "isbn" : [ "9783831642779" ],
+  "language" : [ {
+    "id" : "http://id.loc.gov/vocabulary/iso639-2/deu",
+    "label" : "Deutsch"
+  } ],
+  "medium" : [ {
+    "id" : "http://rdaregistry.info/termList/RDAproductionMethod/1010",
+    "label" : "Print"
+  } ],
+  "natureOfContent" : [ {
+    "id" : "http://d-nb.info/gnd/4113937-9",
+    "label" : "Hochschulschrift"
+  } ],
+  "publication" : [ {
+    "location" : "München",
+    "publishedBy" : "Herbert Utz Verlag",
+    "startDate" : "2017",
+    "type" : [ "PublicationEvent" ]
+  } ],
+  "responsibilityStatement" : [ "Irene R. Kupferschmied" ],
+  "sameAs" : [ {
+    "id" : "http://hub.culturegraph.org/resource/HBZ-HT019268984",
+    "label" : "Culturegraph Ressource"
+  } ],
+  "seeAlso" : [ {
+    "id" : "http://digitool.hbz-nrw.de:1801/webclient/DeliveryManager?pid=7159319&custom_att_2=simple_viewer",
+    "label" : "Digitool"
+  }, {
+    "id" : "http://digitool.hbz-nrw.de:1801/webclient/DeliveryManager?pid=7159320&custom_att_2=simple_viewer",
+    "label" : "Digitool"
+  } ],
+  "subject" : [ {
+    "id" : "http://dewey.info/class/839/",
+    "label" : "Andere germanische Literaturen"
+  }, {
+    "id" : "http://dewey.info/class/230/",
+    "label" : "Christentum, Christliche Theologie"
+  }, {
+    "id" : "http://dewey.info/class/439/",
+    "label" : "Andere germanische Sprachen"
+  }, {
+    "componentList" : [ {
+      "id" : "http://d-nb.info/gnd/4120126-7",
+      "label" : "Altnordisch",
+      "type" : [ "SubjectHeading" ]
+    }, {
+      "id" : "http://d-nb.info/gnd/4199881-9",
+      "label" : "Marienlegende",
+      "type" : [ "SubjectHeading" ]
+    }, {
+      "label" : "Geschichte 1200-1550",
+      "type" : [ "SubjectHeading" ]
+    } ],
+    "type" : [ "ComplexSubject" ]
+  } ],
+  "subjectAltLabel" : [ "Maria / Legende", "Altnordische Sprache", "Marienmirakel" ],
+  "subjectChain" : [ "Altnordisch | Marienlegende | Geschichte 1200-1550" ],
+  "subjectOrder" : [ "http://d-nb.info/gnd/4120126-7, http://d-nb.info/gnd/4199881-9, Geschichte 1200-1550" ],
+  "tableOfContents" : [ {
+    "id" : "http://digitool.hbz-nrw.de:1801/webclient/DeliveryManager?pid=7159319&custom_att_2=simple_viewer",
+    "label" : "Digitool"
+  } ],
+  "title" : "Die altisländischen und altnorwegischen Marienmirakel, 1",
+  "type" : [ "BibliographicResource", "Book", "Thesis" ],
+  "volumeIn" : [ {
+    "multiVolumeWork" : [ {
+      "id" : "http://lobid.org/resources/HT019268980#!",
+      "label" : "lobid Ressource"
+    } ],
+    "numbering" : "1",
+    "type" : [ "MultiVolumeWorkRelation" ]
+  } ]
+}
```

Comment:
Why adding this resource to the test set? Does it really test something that is not already tested by an existing test document?